crates/typst/src/text/raw.rs

```diff
@@ -30,6 +30,7 @@ use crate::{syntax, World};
 type StyleFn<'a> =
     &'a mut dyn FnMut(usize, &LinkedNode, Range<usize>, synt::Style) -> Content;
 type LineFn<'a> = &'a mut dyn FnMut(usize, Range<usize>, &mut Vec<Content>);
+type ThemeArgType = Smart<Option<EcoString>>;
```

Comment:
You might just want to mention in the docs that setting the `theme=none` disables syntax highlighting.

Reply:
Thanks. I have mentioned it in the docs.